apps/web/modules/auth/verify/page.tsx

```
@@ -0,0 +1,16 @@
+import { FormWrapper } from "@/modules/auth/components/form-wrapper";
+import { SignIn } from "@/modules/auth/verify/components/sign-in";
+import { getTranslations } from "next-intl/server";
+
+export const VerifyPage = async (props) => {
+  const searchParams = await props.searchParams;
+  const t = await getTranslations();
+  return searchParams && searchParams.token ? (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)